src/bindings/python/requirements.txt

```diff
@@ -1,3 +1,3 @@
 numpy>=1.16.6
 singledispatchmethod; python_version<'3.8'
-openvino-telemetry>=2023.1.0
+openvino-telemetry>=2023.2.0
```

Comment:
will we provide that 2023.2 openvino package will be future-compatible with `openvino-telemetry`?

Reply:
Yes, it was agreed to make backward-compatible releases of telemetry. Currently there are no plans of significant interface changes except from adding new parameters to constructor. All old parameters are kept.

Reply:
I think it makes sense to add this note and warning that we should not remove current parameters from constructor and add only optional parameters to the constructor or add new constructor interface.
This relates to the whole external API of telemetry.